src/main/scala/mesosphere/marathon/api/v2/serialization/ContainerSerializer.scala

```diff
@@ -0,0 +1,165 @@
+package mesosphere.marathon.api.v2.serialization
```

Comment:
Moved to api. We should consider moving the models from v2 up to api as well. I don't want to incorporate that change into this PR, though.